nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -980,6 +980,25 @@
         - "osgeo._ogr"
         - "osgeo._osr"
 
+- module-name: "panda3d"
+  dlls:
+    - dest_path: "."
+      from_filenames:
+        prefixes:
+        - "libpandagl"
```

Comment:
Are these also for Windows? In that case a `when` clause might be missing.

Reply:
The only one that is definitely windows only is libpandadx9. The rest are cross platform. (But can be disabled by people who do unofficial builds)